### DISCUSSION TITLE
how to get github weeky or monthly  news email ?

### DISCUSSION THREAD

**Discussion by q2333gh** [community/community#72987] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I want to be part of opensource world and exploring it.

**Comment by TankEngine-ish** [comment, marked as answer]
You can sign-up for GitHub's newsletter right here - https://resources.github.com/newsletter/ 
It's once a month.

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @q2333gh 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. As @TankEngine-ish mentioned, you can sign-up for GitHub's newsletter using the following URL: https://resources.github.com/newsletter/.

I also recommend checking out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck on your GitHub journey! 🚀

**Reply by q2333gh** [reply]
so friendly . thank you!
